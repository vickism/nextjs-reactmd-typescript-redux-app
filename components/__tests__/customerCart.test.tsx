import { shallow } from "enzyme";
import React from "react";
import CustomerCard from "../customerCard";

describe("Customer Card Component", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<CustomerCard title="Text1" subTitle="Text2" />);
    expect(wrapper).toMatchSnapshot();
  } )
})