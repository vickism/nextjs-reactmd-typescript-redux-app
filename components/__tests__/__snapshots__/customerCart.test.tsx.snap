// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer Card Component matches snapshot 1`] = `
<div>
  <ForwardRef(Card)>
    <ForwardRef(CardTitle)>
      Text1
    </ForwardRef(CardTitle)>
    <ForwardRef(CardSubtitle)>
      Text2
    </ForwardRef(CardSubtitle)>
  </ForwardRef(Card)>
</div>
`;
